.changeset/clean-geese-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@withease/web-api': patch
```

Comment:
🚨 why it is considered as a patch? As I understand, it is a new feature, and it has to be released as `minor`.